pkg/controller/volume/expand/expand_controller.go

```diff
@@ -282,11 +282,6 @@ func (expc *expandController) syncHandler(ctx context.Context, key string) error
 	if err != nil || volumePlugin == nil {
 		msg := fmt.Errorf("didn't find a plugin capable of expanding the volume; " +
 			"waiting for an external controller to process this PVC")
-		eventType := v1.EventTypeNormal
```

Comment:
Can we keep just "waiting for an external-resizer to process this PVC" ? What if there is no resizer / the resizer is buggy?

Reply:
I have been thinking about it and I think we should be consistent about it with other side cars, otherwise it just confuses users IMO. Do we log similar message with other side cars?

Reply:
We have an event "waiting for a volume to be created, either by external provisioner "csi.vsphere.vmware.com" or manually created by system administrator"